[14.0][MIG] stock_packaging_usability and stock_packaging_usability_pp

alexis-via opened the pull request:
Migration from v12 to v14. Start point: https://github.com/OCA/stock-logistics-tracking/pull/23

Commit:
Add modules stock_packaging_usability and stock_packaging_usability_ul

Commit:
Move module description in README.rst

Commit:
Take into account the remarks of Yannick: self[0] -> self, updates in fr.po

yvaucher committed:
Add bug tracker link on README.rst

alexis-via committed:
Add OCA as author in __openerp__.py

sbidoul committed:
prefix versions with 8.0

pedrobaeza committed:
Make modules uninstallable

pedrobaeza committed:
Rename manifest files

alexis-via committed:
Port stock_packaging usability to v10

alexis-via committed:
stock_packaging_usability: improve README.rst

alexis-via committed:
[MIG] stock_packaging_usability from v10 to v12

Add module stock_packaging_usability_pp

alexis-via committed:
Add fields on product.packaging, in particular the empty weight of the packaging

Fix Odoo problem with weight computation https://github.com/odoo/odoo/issues/34702

alexis-via committed:
Remove debug print

alexis-via committed:
[MIG] stock_packaging_usability and stock_packaging_usability_pp from v12 to v14

alexis-via committed:
stock_packaging_usability*: black, isort and other auto-reformatting

rousseldenis commented:
/ocabot rebase

rousseldenis commented on stock_packaging_usability_pp/wizard/stock_select_product_packaging.py:
```diff
@@ -0,0 +1,39 @@
+    def validate(self):
+        self.ensure_one()
+        default_packaging_id = self.packaging_id.id or False
+        assert (
```
@alexis-via We are in an Odoo context and not in a pure python environment.

IMHO it's better to use UserError. Same for all

rousseldenis commented:
@alexis-via Could you rebase this ?

alexis-via committed:
stock_packaging_usability: FIX bug on reserved_quantity of stock.quant

alexis-via committed:
stock_packaging_usability: try to minimize the impact of the web framework bug

The web framework bug we are trying to mitigate: when we have a button
in tree view that has an attrs/invisible condition: when the conditions
is met after an edit, the columns are not immediately reoganised to
create space for the column.
